packages/server/src/modules/user/mutation/UserRegisterWIthMutation.js

```diff
@@ -0,0 +1,62 @@
+import { GraphQLString, GraphQLNonNull } from "graphql";
+import { mutationWithClientMutationId } from "graphql-relay";
+
+import { generateToken } from "../../../auth";
+// import pubSub, { EVENTS } from '../../../pubSub';
+
+import UserModel from "../UserModel";
+
+export default mutationWithClientMutationId({
+  name: "UserRegisterWithEmail",
+  inputFields: {
+    name: {
+      type: new GraphQLNonNull(GraphQLString)
+    },
+    email: {
+      type: new GraphQLNonNull(GraphQLString)
+    },
+    password: {
+      type: new GraphQLNonNull(GraphQLString)
+    }
+  },
+  mutateAndGetPayload: async ({ name, email, password }) => {
+    let user = await UserModel.findOne({ email: email.toLowerCase() });
+
+    if (user) {
+      return {
+        error: "Email already in use"
+      };
+    }
+
+    if (!name | !email | !password) {
+      return {
+        error: "All fields are required"
```

Comment:
don't need this
GraphQL will throw an error since you're using `GraphQLNonNull` on all of them:
https://github.com/Markkop/fullstacking/blob/2372dce05ef22e6a067efed96a2fa7aacbccd7d4/packages/server/src/modules/user/mutation/UserRegisterWIthMutation.js#L11-L21